ocp_install_env.sh

```diff
@@ -258,10 +258,15 @@ EOF
   fi
 }
 
+function openshift_sdn_deprecated() {
+  # OpenShiftSDN is deprecated in 4.15 and later
+  [[ "${NETWORK_TYPE}" = "OpenShiftSDN" ]] && printf '4.15\n%s\n' "$(openshift_version)" | sort -V -C
+}
```

Comment:
`openshift_sdn_deprecated` makes it sound like this just checks if openshift-sdn is deprecated, but it actually checks if sdn is deprecated _and_ it's the selected `NETWORK_TYPE`.